OpenSim/Common/Test/testTRCFileAdapter.cpp

```diff
@@ -86,22 +88,20 @@ void compareFiles(const std::string& filenameA,
             std::string tokenB{ tokensB[i] };
             OpenSim::IO::TrimWhitespace(tokenA);
             OpenSim::IO::TrimWhitespace(tokenB);
-            tokenA = OpenSim::IO::Lowercase(tokenA);
-            tokenB = OpenSim::IO::Lowercase(tokenB);
             if (tokenA == tokenB) {
                 continue;
             }
-            // We interpreted blank as NaN now revert
-            // to compare to the original file with blanks
-            if (tokenB == "nan") {
-                tokenB = "";
-            }
-            else if (tokenA == "nan") {
-                tokenA = "";
+            tokenA = OpenSim::IO::Lowercase(tokenA);
+            tokenB = OpenSim::IO::Lowercase(tokenB);
+            // We interpreted blank as NaN now make sure
+            // to compare to interpret original file with blanks
+            if (tokenB == "") {
+                tokenB = "nan";
             }
-            if (tokenA == tokenB) {
-                continue;
+            else if (tokenA == "") {
```

Comment:
Sorry if I  missed your response; why do you expect only `tokenA` or `tokenB` to be empty but not both?

Reply:
Sorry; ignore this.